src/notify/event-renderers/prime-eth-withdrawals.ts

```diff
@@ -0,0 +1,47 @@
+import { EmbedBuilder } from 'discord.js'
+import { round } from 'lodash'
+
+import * as lrtDepositPoolAbi from '../../abi/lrt-deposit-pool'
+import { DiscordOptions, notifyDiscord } from '../discord'
+import { registerEventRenderer } from './renderers'
+
+registerEventRenderer(
+  [
+    lrtDepositPoolAbi.events.WithdrawalRequested.topic,
```

Comment:
This feels a bit repetitive, so we add the `topic` here and then an `if` on the `topic` again.

It would be cool if this were a dictionary and the if was done before... just brainstorming ideas

Reply:
Oh it actually could be two separate registers for each topic and then the if isn't necessary at all. The registration should probably not be an array.